LOG.tbasicu

```diff
@@ -27,22 +27,23 @@ uses "File"
 #Region "Equates"
 
 ' Log equates
-
-%LT_Debug = 1 as Long 'Display only if isDebug = %TRUE
-%LT_Time = 2 as Long ' Prepend DateTime$
-%LT_ToConsole = 4 as Long
-%LT_ToConsoleTitle = 8 as Long
-%LT_ToMsgBox = 16 as Long
-%LT_ToLogFile= 32 as long
-%LT_ToStatusBar= 64 as long
-'%LT_ToRichedit = 128 as Long ?
+begin Const
+  %LT_Debug          = 1 'Display only if isDebug = %TRUE
+  %LT_Time           = 2 ' Prepend DateTime$
+  %LT_ToConsole      = 4
+  %LT_ToConsoleTitle = 8
+  %LT_ToMsgBox       = 16
+  %LT_ToLogFile      = 32
+  %LT_ToStatusBar    = 64
+  '%LT_ToRichedit = 128
+end const
 
 $FILE_ROTATION_START_MESSAGE = "Log file rotation * START *."
-$FILE_ROTATION_DONE_MESSAGE = "Log file rotation * DONE *."
-$FILE_BACKUP_MESSAGE = "This Log file backup..."
+$FILE_ROTATION_DONE_MESSAGE  = "Log file rotation * DONE *."
+$FILE_BACKUP_MESSAGE         = "This Log file backup..."
 $FILE_BACKUP_SUCCESS_MESSAGE = "Successful backup of previous Log file as {1}"
-$FILE_ROTATION_TAG = "[LOG UNIT]"
-$FILE_CREATION_MESSAGE = "Creation of Log file {1}"
+$FILE_ROTATION_TAG           = "[LOG UNIT]"
+$FILE_CREATION_MESSAGE       = "Creation of Log file {1}"
 
```

Comment:
Feel free to reject alignment of $EQUATES above, but it is easier to read the texts if they start at the same offset.

Reply:
no, it's good, but this should be a job for thinAir, unfortunately we don't have the hands on the auto formatting params.

Reply:
Believe it or not, loong time ago, around 2005/2006, thinAir had a tool to format code.

It was very nice. I would also like to reappear.

Reply:
Currently there is only 'indent code feature' (also with reported bugs [1](https://www.thinbasic.com/community/project.php?issueid=559), [2](https://www.thinbasic.com/community/project.php?issueid=560), [3](https://www.thinbasic.com/community/project.php?issueid=578), among [many others](https://www.thinbasic.com/community/project.php?projectid=2). Maybe if you could confirm or vote for them , it  would lend them weight. I may be wrong, but I'm guessing Eros had worked on some of them... or had the same thoughts. Anyway, in the last updates, he had worked to give more possibilities to interact with thinAir, that's super.) Back on your $equates alignment : of course it is approved, note that from my side, _manually_ indenting/aligning is challenging.

Reply:
I will talk to Eros regarding this possibility 👍